src/main/java/lotto/domain/ProfitCalculator.java

```diff
@@ -0,0 +1,29 @@
+package lotto.domain;
+
+import java.util.Map;
+import lotto.service.PurchaseService;
+
+public class ProfitCalculator {
+    private final int PERCENTAGE = 100;
+
+    public double calculateProfitRateInPercentage(Map<Rank, Integer> result) {
+        long usedMoney = calculateUsedAmountOfMoney(result);
+        long earnedMoney = calculateEarnedMoney(result);
+        return (double) earnedMoney / usedMoney * PERCENTAGE;
+    }
+
+    private long calculateUsedAmountOfMoney(Map<Rank, Integer> result) {
+        int lottoCount = result.keySet()
+                .stream()
+                .map(result::get)
+                .reduce(0, Integer::sum);
```

Comment:
그냥 sum 함수를 써도 되었을 것 같습니다

Reply:
```suggestion
        int lottoCount = result.keySet()
                .stream()
                .mapToInt(result::get)
                .sum();
```
이미 있는 `sum`을 사용하는 게 나았겠네요 👍🏼